### PR TITLE
Make it clear that the number being logged is a running total

### PR DESCRIPTION
**Pull request by alxndrsn**

### CODE REVIEW

File: transitions/index.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ var processChange = function(change, callback) {
         }
 
         if (processed > 0 && (processed % PROGRESS_REPORT_INTERVAL) === 0) {
-            logger.info('transitions: %d items processed', processed);
+            logger.info('transitions: %d items processed', processed, '(total)');
```
Comment (mandric) [inline transitions/index.js]: This still isn't very clear to me unfortunately...

Comment (mandric) [inline transitions/index.js]: Also is it a pain to use the printf style format like in the previous log line?

Comment (alxndrsn) [inline transitions/index.js]: > This still isn't very clear to me unfortunately...

Any suggestions for improving the message?

> Also is it a pain to use the printf style format like in the previous log line?

Good call - will fix.

Comment (mandric) [inline transitions/index.js]: To me it looks like it would print:

Comment (alxndrsn) [inline transitions/index.js]: Improvement(?) at https://github.com/medic/medic-sentinel/pull/126

**Pull request merged**